projects/tasks.py

```diff
@@ -55,27 +55,25 @@ def end_daypasses():
             )
             invitation.status = Invitation.STATUS_BEYOND_DURATION
             invitation.save()
-        except Exception:
-            LOG.error(f"Error ending daypass invite {invitation.id}")
 
-        try:
-            # If this invite was on a day pass
-            day_pass_request = DayPassRequest.objects.get(invitation=invitation)
-            # And there are 10 requests on that same artifact
-            approved_requests = (
-                DayPassRequest.objects.all()
-                .filter(
-                    artifact=day_pass_request.artifact,
-                    status=DayPassRequest.STATUS_APPROVED,
-                    invitation__status=Invitation.STATUS_BEYOND_DURATION,
+            try:
+                day_pass_request = DayPassRequest.objects.get(invitation=invitation)
+                approved_requests = (
+                    DayPassRequest.objects.all()
+                    .filter(
+                        artifact=day_pass_request.artifact,
+                        status=DayPassRequest.STATUS_APPROVED,
+                        invitation__status=Invitation.STATUS_BEYOND_DURATION,
+                    )
+                    .count()
                 )
-                .count()
-            )
-            if approved_requests == 10:
-                # Send an email
-                handle_too_many_day_pass_users(day_pass_request.artifact)
-        except DayPassRequest.DoesNotExist:
-            pass
+                if approved_requests == settings.DAY_PASS_LIMIT:
```

Comment:
```suggestion
                if approved_requests >= settings.DAY_PASS_LIMIT:
```

Logically, this change is unnecessary, but I think it's better to have this still be handled if some bug in the future somehow allows people to approve users past this limit. With the current logic, if you managed to somehow get over 10 approved requests, you would be allowed to approve all future requests ad-infinitum.

Reply:
Since the allocation is expired at 10, they will have to re-request one which makes sure we are aware of their usage. This lets us gauge the popularity of day pass for now. If we did >=, they would need to request an allocation for every new user after this initial allocation.